RxExample/RxExample/Examples/APIWrappers/APIWrappersViewController.swift

```diff
@@ -139,7 +139,7 @@ class APIWrappersViewController: ViewController {
 
         // also test two way binding
         let textValue = Variable("")
-        _ = textField.rx.textInput <-> textValue
+        (textField.rx.textInput <-> textValue).disposed(by: disposeBag)
```

Comment:
Hi @carlosypunto ,

after our talk, I think that previous version should be ok for now. Could you please revert these changes, so I can merge this?